test/utils/image/manifest.go

```diff
@@ -216,7 +216,7 @@ const (
 
 func initImageConfigs(list RegistryList) (map[ImageID]Config, map[ImageID]Config) {
```

Comment:
there is also `agnhost` mentioned here: `hack/testdata/pod-with-large-name.yaml`

Reply:
not sure if there are more mentions